stripe/src/main/java/com/stripe/android/paymentsheet/repositories/PaymentMethodsRepository.kt

```diff
@@ -15,22 +17,38 @@ internal sealed class PaymentMethodsRepository {
         type: PaymentMethod.Type
     ): List<PaymentMethod>
 
+    abstract suspend fun save(
+        customerConfig: PaymentSheet.CustomerConfiguration,
+        paymentMethodCreateParams: PaymentMethodCreateParams,
+    ): PaymentMethod
+
     /**
      * A [PaymentMethodsRepository] that uses a pre-determined list of payment methods.
      */
     class Static(
         private val paymentMethods: List<PaymentMethod>
     ) : PaymentMethodsRepository() {
+        var error: Throwable? = null
         override suspend fun get(
             customerConfig: PaymentSheet.CustomerConfiguration,
             type: PaymentMethod.Type
         ): List<PaymentMethod> = paymentMethods
+
+        override suspend fun save(
+            customerConfig: PaymentSheet.CustomerConfiguration,
+            paymentMethodCreateParams: PaymentMethodCreateParams
+        ): PaymentMethod {
+            // TODO(michelleb-stripe): What would be the proper payment method to return?
```

Comment:
@mshafrir if you have advice on this one.  Unfortunately I couldn't move it out of the class without exposing a public constructor for PaymentMethodsRepository

Reply:
sealed classes must have a private constructor

Reply:
Would this be resolved by the suggestion here? https://github.com/stripe/stripe-android/pull/3452/files#r592491317